gui/src/main/java/io/bisq/gui/main/dao/compensation/CompensationRequestDisplay.java

```diff
@@ -62,12 +69,19 @@ public void createAllFields(String title, double top) {
         linkHyperlinkWithIcon.setManaged(false);
         linkInputTextField.setPromptText(Res.get("dao.compensation.display.link.prompt"));
         requestedBsqTextField = addLabelInputTextField(gridPane, ++gridRow, Res.get("dao.compensation.display.requestedBsq")).second;
-
+        if (feeService != null) {
+            BsqValidator bsqValidator = new BsqValidator(bsqFormatter);
+            bsqValidator.setMinCompensationRequest(feeService.getCreateCompensationRequestFee());
+            requestedBsqTextField.setValidator(bsqValidator);
+        }
```

Comment:
I think here it is again wrong: The requested BSQ are not the BSQ fee.

Reply:
Same check as the one done in CompensationRequestManager. It's better to make sure to have the check here in the UI, but for correctness I think it should also be handled in CompensationRequestManager.